src/main/java/uk/gov/hmcts/cmc/claimstore/services/DirectionsQuestionnaireDeadlineCalculator.java

```diff
@@ -30,21 +30,17 @@ public DirectionsQuestionnaireDeadlineCalculator(
     }
 
     public LocalDate calculateDirectionsQuestionnaireDeadlineCalculator(LocalDateTime respondedAt) {
-        return calculateFirstWorkingDayAndAddOffset(respondedAt, serviceDays + timeForResponseInDays);
-    }
-
-    private LocalDate calculateFirstWorkingDayAndAddOffset(LocalDateTime dateTime, int offset) {
-        LocalDate date = dateTime.toLocalDate();
+        LocalDate date = respondedAt.toLocalDate().plusDays(serviceDays + timeForResponseInDays);
```

Comment:
Sonar complains about this being two `int`s added to satisfy a `long` parameter. I consider it a FP. If it can't be marked that way in sonarcloud.io then I'll change the fields to `long` instead of casting here.